src/endpoints/Newsrooms/types.ts

```diff
@@ -45,6 +45,8 @@ export interface CreateRequest {
      */
     active_theme?: string;
     status?: Newsroom.Status.ACTIVE | Newsroom.Status.INACTIVE;
+    // Populate newsroom with example content
+    with_demo_content?: boolean;
```

Comment:
> I wasn't sure if it should be here since it's specific to our app

I don't understand. The whole SDK is specific to our app :) 

If the API supports it, it should be here.

Reply:
Yeah, you're right. I was thinking about a use-case that can't really happen in production API (missing example content to duplicate).